manifests/interface.pp

```diff
@@ -323,15 +323,45 @@
   case $::osfamily {
 
     'Debian': {
-      if ! defined(Concat['/etc/network/interfaces']) {
-        concat { '/etc/network/interfaces':
-          mode   => '0644',
-          owner  => 'root',
-          group  => 'root',
-          notify => $network::manage_config_file_notify,
+      if $vlan_raw_device {
+        if !defined(Package['vlan']) {
+          package { 'vlan':
```

Comment:
I'm a bit hesitant in managing in this module a package which might be declared elsewhere (even if the if !defined tries to mitigate possible duplicated resources)